clearwater-queue-manager.root/usr/share/clearwater/clearwater-queue-manager/scripts/check_restart_queue_state

```diff
@@ -17,4 +17,12 @@ then
   exit 1
 fi
 
-/usr/share/clearwater/clearwater-queue-manager/env/bin/python /usr/share/clearwater/clearwater-queue-manager/scripts/check_queue_state.py "${management_local_ip:-$local_ip}" "$local_site_name" "apply_config"
+if [ "$use_single_restart_queue" == "Y" ] ; then
+  /usr/share/clearwater/clearwater-queue-manager/env/bin/python /usr/share/clearwater/clearwater-queue-manager/scripts/check_queue_state.py "${management_local_ip:-$local_ip}" "$local_site_name" apply_config
+else
+  for apply_config_key in $( clearwater-etcdctl ls -p --recursive | grep apply_config_ ); do
+    prefix="/clearwater/site1/configuration/apply_config_"
+    apply_config_key=$(apply_config_key#$prefix)
+    /usr/share/clearwater/clearwater-queue-manager/env/bin/python /usr/share/clearwater/clearwater-queue-manager/scripts/check_queue_state.py "${management_local_ip:-$local_ip}" "$local_site_name" $apply_config_key
+  done
```

Comment:
Is this formatted nicely if you have multiple queues (e.g. line breaks between each entry)?